core/src/autogluon/core/hpo/executors.py

```diff
@@ -158,6 +158,7 @@ class RayHpoExecutor(HpoExecutor):
     custom_to_ray_searcher_preset_map = {
         'local_random': 'random',
         'random': 'random',
+        # TODO: Should this be changed to 'auto': 'bayes'?
         'auto': 'random',
     }
```

Comment:
Currently it's impossible to simultaneously train 
- some models with `backend=custom`, `searcher=random`
- other models with `backend=ray`, `searcher=bayes`

Mapping `auto` to `bayes` backend here would enable us to do that with
```
hyperparameter_tune_kwargs={
    "scheduler": "local",
    "searcher": "auto",
    "num_trials": 10,
}
""

Reply:
@yinweisu